src/Math-Matrix/PMMatrix.class.st

```diff
@@ -102,10 +102,10 @@ PMMatrix class >> lupCRLCriticalDimension [
 ]
 
 { #category : #'instance creation' }
-PMMatrix class >> new: anInteger [
+PMMatrix class >> new: dimension [
```

Comment:
Correct the variable name in the comment, if you don't mind.

Reply:
Yeah, done.